fenix/app/src/main/java/org/mozilla/fenix/onboarding/OnboardingFragment.kt

```diff
@@ -224,8 +224,7 @@ class JunoOnboardingFragment : Fragment() {
         showNotificationPage: Boolean,
         showAddWidgetPage: Boolean,
     ): List<OnboardingPageUiData> {
-        val junoOnboardingFeature = FxNimbus.features.junoOnboarding.value()
-        val jexlConditions = junoOnboardingFeature.conditions
+        val jexlConditions = FxNimbus.features.junoOnboarding.value().conditions
```

Comment:
Should we follow up with renaming the Nimbus feature? I believe this will help if we do it sooner since a year from now, people will be wondering what is juno.

Reply:
Should we have a follow ticket for removing 'juno' from Nimbus, where we capture this & the need to update the [Experimenter](https://experimenter.info/onboarding/onboarding-feature-android/) docs?

Reply:
Awesome 🙌 

>The experimenter docs only need updating if we do decide to update the string keys, right?

Yes that's right! 

>Can we simply update the string keys

Good question, I honestly don't know the answer to this, @Delphine would you be able to provide some guidance on this please? e.g. The actual strings text stays the same but the key/reference would be renamed

If the answer is yes, then it seems like it'd be a nice to have - if not, then I think we leave them as-is, wdyt?

Reply:
When string IDs change, that invalidates existing translations - so I would suggest keeping the string ID the same if the source string doesn't change.